libs/sdk-backend-spi/api/sdk-backend-spi.api.md

```diff
@@ -836,6 +844,8 @@ export interface IFilterElementsQuery {
 
 // @alpha
 export interface IGetDashboardOptions {
+    // (undocumented)
+    exportId?: string;
```

Comment:
Pls can you document this item ?

Reply:
fixed

Reply:
actually I've documented in 48ced1ac556acf2c92b3f0f9df86fb070e23eae0 hopefully that's sufficient, the docs do not get here to this exact place.